AppTaskQueue/appscale/taskqueue/distributed_tq.py

```diff
@@ -616,6 +641,22 @@ def get_task_args(self, request):
       args['max_backoff_sec'] = queue.max_backoff_seconds
       args['max_doublings'] = queue.max_doublings
 
+      # If we could not get the target from the host, try to get it from the
+      # queue config.
```

Comment:
Shouldn't it be whatever target was specified as, then queue config, then current running?